CHANGELOG.md

```diff
@@ -6,6 +6,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- When using `-v` verbose flag, will set `debug` as logging level, ignoring any other passed.
+
+### Fixed
+
+- If using docker image, and `info` logging level, it will be now used instead of `debug`.
```

Comment:
Can you add the reference to the issue here?
Lately we have a lot of release notes without any issue reference and it makes it quite hard to expand info about each item.

Reply:
True! I always did it in Landing, and forgot it here.
Done by 52f548e